src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <div id="dialog">
+        <modal :open="open">
+            <h4 v-translate="'uploadFileToAdr'"></h4>
+            <div class="container">
+                <div id="dataset-id" class="mt-4">
+                    <span v-translate="'uploadFileDataset'"></span>
+                    <span>{{ dataset }}</span></div>
+                <div id="instructions" class="mt-3" v-translate="'uploadFileInstruction'"></div>
+                <div id="output-file-id" class="mt-3" v-for="uploadFile in uploadFiles" :key="uploadFile.index">
```

Comment:
I think you should be iterating in order of the `index` field of the values in `uploadFiles` - it that right @EmmaLRussell?

Reply:
I don't mind changing this to the display name but I believe Emma wanted it to be index.